packages/server/src/sdk/app/backups/backup.ts

```diff
@@ -0,0 +1,145 @@
+import { backups } from "@budibase/pro"
+import { db as dbCore, objectStore, tenancy } from "@budibase/backend-core"
+import {
+  AppBackupQueueData,
+  AppBackupStatus,
+  AppBackupTrigger,
+  AppBackupType,
+} from "@budibase/types"
+import { exportApp } from "./exports"
+import { importApp } from "./imports"
+import { calculateBackupStats } from "../statistics"
+import { Job } from "bull"
+import fs from "fs"
+import env from "../../../environment"
+
+type BackupOpts = {
+  doc?: { id: string; rev: string }
+  createdBy?: string
+}
+
+async function removeExistingApp(devId: string) {
+  const devDb = dbCore.dangerousGetDB(devId, { skip_setup: true })
```

Comment:
Could use a `try`-`finally` here to ensure the destroy is called

Reply:
Sorry - not quite sure what you mean, if an error occurs in the `dbCore.dangerousGetDB` then there won't be anyway to destroy the db anyway - do you mean in the import processor?

Reply:
Ah sorry, I mean for a `db.close`. Do we need to add one here, or will destroy auto close?

Reply:
it auto closes in these scenarios, generally we don't need to worry about it unless you were to open it in a global context! But it'll close automatically when garbage collected.